### PR TITLE
Working on (funcionar em) android devices.

### PR DESCRIPTION
**Pull request by pedroso**
PT-BR
Alteração para corrigir compatibilidade com o Chrome em dispositivos Android;
EN
Change to fix compatibility with Chrome on Android devices;

### CODE REVIEW

**Comment by PowerKiKi** [comment]
Please change the commit message to be english only, and I'll merge

**Comment by NickTomlin** [comment]
@pedroso could you also give some more context on why this is needed? I don't doubt that it is a fix, i'm just not aware of what behavior it is fixing (e.g. is this _all_ versions of chrome on Android or just legacy ones?).

**Comment by pedroso** [comment]
Ok man. Sorry, that is my first!

I'll change...

**Comment by pedroso** [comment]
The mask does't work on chrome version 42 in android devide. I tested and
the position of the character stops on the second position.

**Comment by pedroso** [comment]
Sorry about my english...

**Comment by NickTomlin** [comment]
Okay that sounds good. I haven't tested this on android but it looks good to me.

File: dist/mask.js [hunk 1]
```diff
@@ -459,10 +462,14 @@ angular.module('ui.mask', [])
                                 return maskCaretMap.indexOf(pos) > -1;
                             }
 
-                            function getCaretPosition(input) {
+                            function getCaretPosition(input, eventWhich) {
                                 if (!input)
                                     return 0;
+                                var isRemoving = eventWhich === 8 || eventWhich === 46;
```
Comment (NickTomlin) [inline dist/mask.js]: If you have a moment could you explain how this is functioning for android? It's not a blocker; i'm just curious :)

**Comment by pedroso** [comment]
Ok bro. Thanks. I tested on Sansung galaxy tab 4.

**Comment by NickTomlin** [comment]
@PowerKiKi I think this is good to go. I will try to add automated testing for mobile devices soon (I only have a iOs device at home).

**Comment by PowerKiKi** [comment]
Thanks to you both @pedroso and @NickTomlin

**Comment by PowerKiKi** [comment]
Released as 1.4.7

**Comment by pedroso** [comment]
You're welcome!

**Pull request merged**